src/Components/ArticlesList.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import ArticleCard from "./ArticleCard"; 
+
+
+const ArticlesList = () => {
+  const [articles, setArticles] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    axios.get("https://nc-news-ts.onrender.com/api/articles")
+      .then((response) => {
+        setArticles(response.data.articles);
+        setLoading(false);
+      })
+      .catch((error) => {
+        console.error("Error fetching articles:", error);
```

Comment:
Instead of logging here you can display an error message to the user using conditional rendering.